agent.go

```diff
@@ -1210,3 +1244,59 @@ func winsizeCb(pod *pod, data []byte) error {
 
 	return nil
 }
+
+//Get the agent configuration from kernel cmdline
+func getConfig() agentConfig {
+
+	config := agentConfig{}
+	config.logLevel = defaultLogLevel
```

Comment:
I think it would be better if this function also returned an `error`. The caller can then set `config.logLevel = defaultLogLevel`. It's neat to encapsulate the default handling like this, but it makes testing it difficult (how can you know which codepath was followed if `kernelCmdlineFile` contained the same `defaultLogLevel` value?)

Speaking of which, can you add a unit test for this function :smile: